CleverFerret/src/main/java/com/universalmedialibrary/ui/components/ChapterListDialog.kt

```
@@ -98,96 +83,124 @@ fun ChapterListDialog(
 )
+
+/**
+ * Format duration in milliseconds to readable string
+ */
+private fun formatDuration(millis: Long): String {
+    val seconds = millis / 1000
+    val minutes = seconds / 60
+    val hours = minutes / 60
+    
+    return when {
+        hours > 0 -> String.format("%d:%02d:%02d", hours, minutes % 60, seconds % 60)
```

detekt (1.23.8)

[warning] String.format("%d:%02d:%02d", hours, minutes % 60, seconds % 60) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)